GTG/gtk/browser/taskbrowser.ui

```diff
@@ -397,7 +397,7 @@
                   <object class="GtkToolButton" id="new_subtask_b">
                     <property name="can_focus">False</property>
                     <property name="label" translatable="yes">New Subtask</property>
-                    <property name="icon_name">gtg-task-new</property>
+                    <property name="icon_name">Gtk.STOCK_NEW</property>
```

Comment:
I don't think this is going to work, you need to use the literal value here: `gtk-new`